api/src/main/java/org/ehcache/event/CacheEvent.java

```diff
@@ -19,6 +19,11 @@
 import org.ehcache.Cache;
 
 /**
+ * Class representing an event resulting from a mutative {@link Cache} operation.
```

Comment:
doesn't really matter but you could maybe drop the "Class representing" at the beginning